src/contractTests/prisonerProfile/scenarios/domains/interventions/personInterventions/CSIPReadScenarios.ts

```diff
@@ -1,4 +1,4 @@
-import { baseCheckAndInUsersCaseLoadScenarios } from '../../../shared/BaseCheckAndInUsersCaseLoadScenarios'
+import { inUsersCaseLoadScenarios } from '../../../shared/InUsersCaseLoadScenarios'
 
 // eslint-disable-next-line import/prefer-default-export
-export const csipReadScenarios = baseCheckAndInUsersCaseLoadScenarios
+export const csipReadScenarios = inUsersCaseLoadScenarios
```

Comment:
Just a rename to hightlight the important part of the check (that the user's caseload contains the prisoner's prison)